canopy_article.ml

```diff
@@ -49,3 +49,23 @@ let to_tyxml_listing_entry article =
       br ();
     ] in
   a ~a:[a_href article.uri; a_class ["list-group-item"]] (content ++ abstract)
+
+let to_atom { title; author; abstract; uri; date; tags; content; } =
+  let text x : Syndic.Atom.text_construct = Syndic.Atom.Text x in
+  let summary = match abstract with
+    | Some x -> Some (text x)
+    | None -> None
+  in
+  let categories =
+    List.map
+      (fun x -> Syndic.Atom.category ~scheme:(Uri.of_string ("/tags/" ^ x)) x)
+      tags
+  in
+  Syndic.Atom.entry
+    ~id:(Uri.of_string uri)
+    ~content:(Syndic.Atom.Html (None, content))
+    ~authors:(Syndic.Atom.author author, [])
+    ~title:(text title)
+    ~updated:date
```

Comment:
We can have a published timestamp in `<id>` tag (like `2016-04-05:/20160405194055623` with `timestamp = 20160405194055623`). It's not mandatory to change the `<updated>` tag when you change the article. So if @Engil provides the published timestamp (but I think `irmin`does not provide this information), I can use it in `<updated>` tag.